devtools/conda-envs/test_env.yaml

```diff
@@ -17,34 +17,30 @@ dependencies:
   - tensorboard
   - torchvision
   - openff-units
+  - torchmetrics>=1.4
   - pint
   - rdkit
   - retry
   - sqlitedict
   - pytorch_scatter
-  - jax
 
   # Testing
   - pytest>=2.1
   - pytest-cov
   - codecov
   - requests
-  - ray-data
-  - ray-train
-  - ray-tune
   - torchani
   - versioneer
 
   # Docs
   - sphinx_rtd_theme
 
-  # pip installs
   - pip:
-      - schnetpack>=2.0.0
+      # - schnetpack>=2.0.0
```

Comment:
As we discussed offline, we should probably just move the tests for the original implementation to a separate subdirectory (with its own environment) and not run these with CI, just run before releases.

Reply:
agreed!